Zhongli.Bot/Modules/ChannelModule.cs

```diff
@@ -0,0 +1,380 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Zhongli.Data.Models.Authorization;
+using Zhongli.Services.CommandHelp;
+using Zhongli.Services.Core.Preconditions.Commands;
+using Discord.WebSocket;
+
+namespace Zhongli.Bot.Modules;
+
+[Group("channel")]
+[Name("Channel Management")]
+[Summary("Manages channels and sync permissions.")]
+[RequireBotPermission(GuildPermission.ManageChannels)]
+[RequireUserPermission(GuildPermission.ManageChannels, Group = nameof(ChannelModule))]
+[RequireAuthorization(AuthorizationScope.Channels, Group = nameof(ChannelModule))]
+public class ChannelModule : ModuleBase<SocketCommandContext>
+{
+    private readonly ICommandHelpService CommandHelpService;
+    public ChannelModule(ICommandHelpService commandHelpService)
+    {
+        this.CommandHelpService = commandHelpService;
+    }
+
+
+    /* Create Channel */
+    [Command("create")]
+    [Summary("Creates a new channel.")]
+    public async Task CreateChannelAsync(string name, ChannelCreationOptions? options = null)
+    {
+        // TODO: V2 Add channel creation options.
+        var channel = await Context.Guild.CreateTextChannelAsync(name);
+        var embed = new EmbedBuilder()
+           .WithTitle($"Created Channel \"{name}\" : {channel.Id}")
+           .AddField("Channel ID", channel.Id, true)
+           // .AddField() TODO: v2 Display channel creation options.
+           .WithDescription($"Successfully created channel with name \"{name}.\"")
+           .WithAuthor(Context.User);
+        await Context.Channel.SendMessageAsync(embed: embed.Build());
```

Comment:
You can use `ReplyAsync` which is shorthand for this
```suggestion
        await ReplyAsync(embed: embed.Build());
```